uber/site_sections/accounts.py

```diff
@@ -20,8 +20,7 @@ def index(self, session, message=''):
                                .order_by(Attendee.first_name, Attendee.last_name).all(),
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), c.BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
-                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num).all()
```

Comment:
I'm a little confused about the logic behind this. Maybe there's an issue I missed that discussed the reasoning behind doing this?

It makes sense that we never want to create admin accounts for people without email addresses - we need to email them their password, for example - but why do we include those people in the list of the Accounts page? The user would assume that anyone in the list can have an account, then be disappointed when they get an error. It's a somewhat cumbersome way to go about it - you'd hunt for the right name in the list, have the person enter their password, and _then_ find out that they can't have an account until you go to the Attendees page, find them, set their email, and then go back and find their name in the list of attendees under the Accounts page.

That doesn't prevent us from merging this PR, but it's definitely something to think about. Maybe a quick fix could be to display an email field that you can enter the email in while creating the admin account?